src/ns/UserNamespaceListener.cc

```diff
@@ -41,10 +41,10 @@ void UserNamespaceListener::onPreFork() {
     gid_t gid = rootOutsideGid_;
 
     // watch out, they're unsigned ;)
-    if (uid == (uid_t) -1) {
+    if (uid == static_cast<uid_t>(-1)) {
```

Comment:
IMO to nie jest bardziej czytelne.

Reply:
To akurat to kwestia gustu (np, ja bardziej lubie taki cast, widać dokładniej co jest castowane i na co). Poza tym taki cast ma lepiej zdefiniowaną semantykę.